cmd/kube-apiserver/app/server.go

```diff
@@ -127,6 +127,9 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	// Use protobufs for self-communication.
+	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
I'd rather we auto-detect, but since it would seem we don't, this seems scoped correctly.  I think the common case with multiple apiservers is "not protobuf". 

We have roundtripping tests which make sure that protobuf is turned on for the kube-apiserver's scheme.